build-wasm.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+RELEASE_VERSION=$(cat RELEASE)
+DIR=$(cd $(dirname ${BASH_SOURCE[0]}) && pwd)
+BUILD_DIR="${DIR}/build-wasm"
+DIST_DIR="${DIR}/dist/openjpeg-wasm"
+
+ENVIRONMENT="production";
+BROWSER_OUTPUT="openjpeg"
+OPTIMIZE="-Os"
+if [$ENVIRONMENT != "production"]
+then
+    OPTIMIZE="-O2 --js-opts 0 -g4 --source-map-base http://localhost:8080/"
+fi
+LDFLAGS="${OPTIMIZE}"
+CFLAGS="${OPTIMIZE}"
+CXXFLAGS="${OPTIMIZE}"
```

Comment:
Do these flags actually do anything? We use `$OPTIMIZE` directly later

Reply:
Yeah, they can be cut now, they've been there so long now I don't even see them.